checkov/common/output/report.py

```diff
@@ -398,14 +398,18 @@ def print_failed_github_md(self, use_bc_ids: bool = False) -> str:
                     record.guideline,
                 ]
             )
-        output_data = tabulate(
-            result,
-            headers=["check_id", "file", "resource", "check_name", "guideline"],
-            tablefmt="github",
-            showindex=True,
-        ) + "\n\n---\n\n"
-        print(output_data)
-        return output_data
+        if result:
+            table = tabulate(
+                result,
+                headers=["check_id", "file", "resource", "check_name", "guideline"],
+                tablefmt="github",
+                showindex=True,
+            )
+            output_data = f"### {self.check_type} scan results:\n {table}  \n\n---\n\n"
```

Comment:
```suggestion
            output_data = f"### {self.check_type} scan results:\n\n{table}\n\n---\n"
```
how about doing it this way?